pizzaplace/services/pizza_place_presenter.py

```diff
@@ -0,0 +1,19 @@
+class PizzaPlacePresenter(): 
```

Comment:
You can configure your editor to remove trailing whitespace when you save a file.